Sample_v3/Samples/SimpleChat/SimpleChatViewController.swift

```diff
@@ -334,3 +400,18 @@ extension SimpleChatViewController {
         tableView.transform = CGAffineTransform(scaleX: 1, y: -1)
     }
 }
+
+// MARK: - Private
+
+private extension UITableViewCell {
+    var swipeActionsContainer: UIView? {
+        guard
+            let superview = superview,
+            let swipeContainerViewType = NSClassFromString("_UITableViewCellSwipeContainerView"),
+            let swipeViewType = NSClassFromString("UISwipeActionPullView"),
+            superview.isKind(of: swipeContainerViewType)
+        else { return nil }
+        
+        return superview.subviews.first { $0.isKind(of: swipeViewType) }
+    }
+}
```

Comment:
What kind of sorcery is this? :D

Reply:
Ahaha 😄 That's the only solution I found (with some creativity to make it look nicer) that deals with the following issue:
 
![Simulator Screen Shot - iPhone 8 - 2020-09-07 at 12 16 54](https://user-images.githubusercontent.com/12818985/92371250-0d171080-f104-11ea-911f-dd52553a788a.png)

Reply:
Hmm.. 🤔  I might be wrong, but seems it's not enough to change the message ordering

If we don't mirror the table view the `load more` logic has be adjusted +without flipping the table view we also have to auto-scroll the table to the last cell so that users can see the latest messages first

Reply:
I'm not saying it's the only required change, but it's the change that allows us to make this without any hacks.